screens/Scan/ScanLayout.tsx

```diff
@@ -62,8 +83,26 @@ export const ScanLayout: React.FC = () => {
           name={SCAN_ROUTES.ScanScreen}
           component={ScanScreen}
           options={{
-            headerTitleStyle: {fontSize: 30, fontFamily: 'Inter_600SemiBold'},
             title: t('MainLayout:share'),
+            headerTitle: props => (
+              <View
+                style={{
```

Comment:
Can't we move this style to Theme class?

Reply:
moved to Theme files